network/internal/messageutils/utils.go

```diff
@@ -1,49 +0,0 @@
-package messageutils
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/onflow/flow-go/model/flow"
-	libp2pmessage "github.com/onflow/flow-go/model/libp2p/message"
-	"github.com/onflow/flow-go/network/channels"
-	"github.com/onflow/flow-go/network/message"
-	"github.com/onflow/flow-go/network/p2p"
-	"github.com/onflow/flow-go/utils/unittest"
-)
-
-func CreateMessage(t *testing.T, originID flow.Identifier, targetID flow.Identifier, channel channels.Channel, msg string) (*message.Message, *message.Message, interface{}) {
```

Comment:
With the changes in this PR, we no longer deal directly with the `message.Message` in testing. Rather, we use the `IncomingMessageScope` and `OutgoingMessageScope`.